pandas/core/indexes/accessors.py

```diff
@@ -572,6 +572,44 @@ class PeriodProperties(Properties):
 class CombinedDatetimelikeProperties(
     DatetimeProperties, TimedeltaProperties, PeriodProperties
 ):
+    """
+    Accessor object for Series values' datetime-like, timedelta and period properties.
+
+    See Also
+    --------
+    Series.dt : Accessor object for datetimelike properties of the Series values.
```

Comment:
This is the own accessor being documented. Maybe we can add `DatetimeIndex` here.